typescript/cli/package.json

```diff
@@ -45,13 +45,16 @@
     "eslint-plugin-import": "^2.31.0",
     "mocha": "^10.2.0",
     "prettier": "^2.8.8",
-    "typescript": "5.3.3"
+    "typescript": "5.3.3",
+    "webpack": "^5.98.0",
+    "webpack-cli": "^6.0.1"
   },
   "scripts": {
     "hyperlane": "node ./dist/cli.js",
-    "build": "yarn version:update && tsc",
+    "build": "yarn version:update && tsc && yarn bundle",
+    "bundle": "rm -rf ./bundle && webpack --config webpack.config.mjs",
```

Comment:
do we need to bundle on every build? can we just call `yarn bundle` as part of the `yarn pack` CI step?

Reply:
yes, to update the bundled build so that when we install the cli we always have the last version